angular-chess/src/app/components/chess-field/chess-field.component.html

```diff
@@ -15,7 +15,7 @@
     class="field">
     <div
       *ngIf="piece"
-      class="piece color-{{ square?.highlight }}"
+      class="piece color-{{ getHighlightColorRepresentation(square?.highlight) }}"
```

Comment:
you can create a public const for this color setting to remove this function call in template. 
Or try this out this signals.